jolie/src/jolie/OOITBuilder.java

```diff
@@ -1384,22 +1239,16 @@ public void visit( InstanceOfExpressionNode n )
 	public void visit( TypeCastExpressionNode n )
```

Comment:
Looks like the commit is reverting my changes to this method for some reason. The correct version is the one with the switch.